src/RootCloseWrapper.js

```diff
@@ -4,6 +4,14 @@ import EventListener from './utils/EventListener';
 
 // TODO: Merge this logic with dropdown logic once #526 is done.
 
+const clickWasInside = Symbol('clickWasInside');
+
```

Comment:
Oh crap. I thought we were already using `babel-runtime`. We're not. This can't be merged as-is, then.